src/main/java/edu/wpi/first/gradlerio/deploy/roborio/FRCJavaArtifact.java

```diff
@@ -116,12 +118,26 @@ public FRCJNILibraryArtifact getNativeZipArtifact() {
         return arguments;
     }
 
+    public void useDefaultJvmArgs(boolean useDefaultJvmArgs) {
+        this.useDefaultJvmArgs = useDefaultJvmArgs;
+    }
+
     private String generateStartCommand(DeployContext ctx) {
         StringBuilder builder = new StringBuilder();
-        builder.append("/usr/local/frc/JRE/bin/java -XX:+UseConcMarkSweepGC -Djava.library.path=");
+        builder.append("/usr/local/frc/JRE/bin/java");
+        builder.append(" -Djava.library.path=");
         builder.append(FRCDeployPlugin.LIB_DEPLOY_DIR);
-        builder.append(" -Djava.lang.invoke.stringConcat=BC_SB ");
-        builder.append(String.join(" ", jvmArgs));
+
+        String jvmArgsString = String.join(" ", jvmArgs);
+        if(useDefaultJvmArgs) {
+            builder.append(" -Djava.lang.invoke.stringConcat=BC_SB ");
```

Comment:
Unlike the -XX option, this can be overridden with an additional -D option.  I would prefer this not be lumped in with that.